packages/config/config/devices/0x015f/mh-s314-1502.json

```diff
@@ -0,0 +1,703 @@
+	"label": "MH-S314",
+	"description": "Touch Panel Switch 4 Button",
+	"devices": [
+		{
+			"productType": "0x3141",
+			"productId": "0x5102",
+			"zwaveAllianceId": 1114
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 1,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Button 1",
+			"maxNodes": 5
+		},
+		"5": {
+			"label": "Button 2",
+			"maxNodes": 5
+		},
+		"8": {
+			"label": "Button 3",
+			"maxNodes": 5
+		},
+		"11": {
+			"label": "Button 4",
+			"maxNodes": 5
+		}
+	},
+	"paramInformation": {
+		"2": {
+			"label": "Recover State After Power Failure",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 0
+				},
+				{
+					"label": "Enabled",
+					"value": 1
+				}
+			]
+		},
+		"3": {
+			"label": "All On/All Off",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 255,
+			"defaultValue": 255,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Forbid All On and All Off",
+					"value": 0
+				},
+				{
+					"label": "Forbid All On",
+					"value": 1
+				},
+				{
+					"label": "Forbid All Off",
+					"value": 2
+				},
+				{
+					"label": "Allow All On and Off",
+					"value": 255
+				}
+			]
+		},
+		"4": {
+			"label": "LED Backlight Brightness Level",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 10,
+			"defaultValue": 10,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"5": {
+			"label": "Key Mode",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 6,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Single click to switch on/off",
+					"value": 0
+				},
+				{
+					"label": "Click to turn on, auto off after param 6 duration",
+					"value": 1
+				},
+				{
+					"label": "Click to turn off, auto on after param 6 duration",
+					"value": 2
+				},
+				{
+					"label": "Hold >3s for on until released",
+					"value": 3
+				},
+				{
+					"label": "Single click to toggle, hold >3s for on until released",
+					"value": 4
+				},
+				{
+					"label": "Momentary, hold for on until released",
+					"value": 5
+				},
+				{
+					"label": "Hold >3s to switch on/off",
+					"value": 6
+				}
+			]
+		},
+		"6": {
+			"label": "Key On Duration",
+			"description": "0 for infinite",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 65535,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"8": {
+			"label": "Basic CC Integration Setting",
```

Comment:
Any idea what this is, or what the modes do? Can we make these labels more descriptive? Add a description to the parameter if not? I am only marking things once so please carry through.